platform/persistence/platform-persistence-core-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,11 +18,13 @@
 
     <reference id="encryptionService" interface="ddf.security.encryption.EncryptionService"/>
 
+    <bean id="solrFactory" class="org.codice.solr.factory.impl.SolrClientFactoryImpl" />
+
     <bean id="persistentStore" class="org.codice.ddf.persistence.internal.PersistentStoreImpl">
         <cm:managed-properties
```

Comment:
Since you removed the metatype file, I think you should remove this manage-properties section.